app/components/AdjustableAnalogClock.qml

```diff
@@ -97,7 +97,7 @@ AnalogMode {
         anchors.centerIn: parent
 
         smooth: true
-        source: "../graphics/Minute_Adjust_Cirlce.png"
+        source: Theme.name == "Ubuntu.Components.Themes.Ambiance" ? "../graphics/Minute_Adjust_Cirlce.png" : "../graphics/Minute_Adjust_Cirlce_White.png"
```

Comment:
It  might  be better to use a lookup table/hash to match the  theme to the associated graphics  but  i don't  think this should hold up the merge.

Reply:
I thought also about adding a filter/shader on that image as an `Icon` would do to change to color from grey to white (#f2f2f2) but It wasn't so easy...

Reply:
Btw, the telegram app (the actual one, not the newest) uses similar code for some theming stuff and imo is a quite good code, easy and efficient